CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 Unreleased
 ------
 - Replace css preprocessor, from [Sass](https://sass-lang.com) to [Less](http://lesscss.org)
+- Upgrade to gulp 4
```

Comment:
another this-pr-is-draft note: release notes will be more thorough, as will commit message and PR description. i have so much to say 😄

Reply:
Just kidding. Any changes are in dev dependencies, this should be sufficient.